### PR TITLE
ja.json

### PR DESCRIPTION
**Pull request by FNJPNews**

### CODE REVIEW

File: ja.json [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+      "title-all": "全てのアイテム",
+      "error": "アイテムを取得できませんでした。しばらくしてからもう一度お試しください。",
+      "error-too-long": "予想よりも長くかかっています。"
+    },
+    "shop": {
+      "title": "現在のアイテムショップ",
+      "title-featured": "イチオシ！",
+      "title-daily": "デイリー",
+      "error": "ショップを取得できませんでした。しばらくしてからもう一度お試しください。",
+      "line1": "フォートナイト バトルロイヤルのショップは毎日[00:00 UTC](https://time.is/UTC|tooltip=Current UTCの時刻|newtab=true)に更新されます。",
```
Comment (Krymonota) [inline ja.json]: Please translate `Current` (and possibly `UTC`?)

File: ja.json [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+      "popular-title": "今日の人気商品"
+    },
+    "account": {
+      "home": {
+        "title": "アカウントデータ",
+        "error": "アカウントデータの読み込み中に問題が発生しました。しばらくしてからもう一度お試しください。",
+        "info": {
+          "title": "情報",
+          "line1": "[provider]で[username]としてログインしています",
+          "line2-api-yes": "[api access](/api/docs)があります",
```
Comment (Krymonota) [inline ja.json]: Please translate `api access`.

File: ja.json [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+          "twitter-line1": "ログインにDiscordを使用するようになりました。現在使用している古いアカウントは将来廃止される可能性があり、機能が制限されています。",
+          "twitter-line2": "Discordアカウントに移行するには、下のボタンをクリックしてください"
+        },
+        "ratings": {
+          "title": "評価",
+          "none": "評価はありません"
+        },
+        "destroy": {
+          "title": "アカウントの破棄",
+          "line1": "アカウントが不要になった場合、アカウントとそのすべてのデータを削除できます[here](/account/destroy)"
```
Comment (Krymonota) [inline ja.json]: Please translate `here`.

File: ja.json [hunk 4]
```diff
@@ -0,0 +1,298 @@
...
+        }
+      },
+      "oauth": {
+        "title": "fnbr.coへのログイン",
+        "discord-button": "Discordでログイン",
+        "legacy-text": "または、移行が必要な2018年の古いアカウントがある場合:",
+        "twitter-button": "Twitterでログイン",
+        "privacy-text": "ログインすると、[privacy policy](/privacy)に記載されているように、アカウントに関連付けられたデータを保存することに同意したことになります。."
```
Comment (Krymonota) [inline ja.json]: Please translate `privacy policy`.

File: ja.json [hunk 5]
```diff
@@ -0,0 +1,298 @@
...
+        "line1": "Discordでログインしています。サイトの他のユーザーとやり取りするには、ユーザー名を選択する必要があります",
+        "nameTaken": "あなたのユーザー名[attemptedName]は別のユーザーに使用されています。名前を変更することをお勧めします。",
+        "randomName": "ユーザー名[attemptedName]は別のユーザーに使用されています。名前を作成できませんでした。",
+        "form-username": "ユーザー名",
+        "form-criteria": "2〜32文字のユーザー名を選択してください。英数字、アンダーバー、ハイフンを含めることができます。",
+        "button-text": "ユーザー名を設定"
+      },
+      "locked": {
+        "title": "ログインできません",
+        "line1": "[your account has been locked](class=text-danger)のため、fnbr.coにログインできません。."
```
Comment (Krymonota) [inline ja.json]: Please translate `your account has been locked`.

File: ja.json [hunk 6]
```diff
@@ -0,0 +1,298 @@
...
+      "migrated": {
+        "title": "移行されたアカウント",
+        "line1": "Twitterでログインしました",
+        "line2": "しかし、そのアカウントはDiscordログインを使用するために移行されました。"
+      },
+      "notFound": {
+        "title": "アカウントが見つかりません",
+        "line1": "Twitterでログインしました",
+        "line2": "しかし、そのアカウントは以前にログインしたことがなく、現在は移行目的でTwitterログインのみをサポートしています。",
+        "line3": "または、[login with Discord](/oauth/discord)することもできます。"
```
Comment (Krymonota) [inline ja.json]: Please translate `login with Discord`.

File: ja.json [hunk 7]
```diff
@@ -0,0 +1,298 @@
...
+    },
+    "errors": {
+      "404": {
+        "title": "エラー",
+        "line1": "リクエストしたリソースが存在しないか、表示する権限がありません。"
+      },
+      "500": {
+        "title": "あぁ！",
+        "line1": "リクエストの処理中に問題が発生しました。しばらくしてからもう一度お試しください！",
+        "line2": "[here](/oauth)で再度ログインを試みることができます。."
```
Comment (Krymonota) [inline ja.json]: Please translate `here`.

**Comment by FNJPNews** [comment]
All corrections have been translated and changed.

**Comment by Krymonota** [comment]
Thanks!
Strings now seem to be fine, JSON is valid and the file has passed the test script. I'll take care of the rest.

**Pull request merged**